### DISCUSSION TITLE
First look and wishes

### DISCUSSION THREAD

**Discussion by NikolayGur** [Moonvy/figmaEX#8] [Ideas]
Good afternoon. 
Thank you for your hard work. Installed the panel, and this is the first thing I want to tell you:
1. The panel will be in demand and popular, but in my opinion needs improvement.
1.1 Add the ability to choose how the panel is set to the screen. (Drag and drop, horizontal and vertical placement options).
1.2 Add ability to resize plugin icons.
1.3 Add ability to remove plug-ins from panel (quick delete) from panel and not from program. 
Make the panel a list of favorite plugins. 
1.4 Add a quick add plugins. (Important if you will fulfill point 1.3).
1.5 Add ability to safely and quickly delete or hide the panel. 

At the moment the panel can only be removed by reinstalling Figma. And that is not without problems. Have to change attributes of the old directory of Figma, which would have to install it again.

**Comment by yArna** [comment]
Very nice feedback,

We recently prepared a grouping feature, to allow users to group the plug-ins into different groups and switch between them quickly, Possible to meet 1.3 1.4 requirements.


About Uninstall,  figmaEX installer has option `2. Restore Backup` for uninstall,  Works fine on our machines, Some people feedback that it is not working, we need more testing.

**Reply by NikolayGur** [reply]
Uninstalling via Restore Backup didn't work for me either. Windows 10 pro. Figma 97.7.
I had to uninstall Figma. Not that I didn't like the panel....))) Just decided to check out all the available features.

**Reply by yArna** [reply]
What is the local language setting of your system?

**Reply by NikolayGur** [reply]
Russian

**Reply by yArna** [reply]
We updated the installer [FigmaEX v1.0.3](https://github.com/Moonvy/figmaEX/releases/tag/v1.0.3) , should be able to uninstall normally with him

**Comment by yellingbytes** [comment]
- Able to dock the freq use plugins
- able to dock the fav plugins
- Drag the panel on top/bottom/left
